examples/virtual_sites/vsite_showcase.ipynb

```diff
@@ -10,17 +10,16 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    positions = to_openmm(
</pre><p>(not blocking) This has become a chonker of a line, can it be broken into a few lines to improve readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>